java/code/src/com/suse/manager/reactor/PGEventStream.java

```diff
@@ -89,7 +97,7 @@ public PGEventStream() throws SaltException {
             startConnectionWatchdog();
 
             LOG.debug("Listening succeeded, making sure there is no event left in queue...");
-            notification(SaltEventFactory.countSaltEvents());
+            notification(SaltEventFactory.countSaltEvents(THREAD_POOL_SIZE));
```

Comment:
Should we use `THREAD_POOL_SIZE + 1` here, and remove the whole "+1" notion in `SaltEventFactory`, so that it remains confined in this class?

Reply:
Yep

Reply:
:heavy_check_mark: